home.html

```diff
@@ -0,0 +1,147 @@
+                      <li class="menu-items"><a href="#" class="menu-list-a">News</a></li>
+                      <li class="menu-items"><a href="#" class="menu-list-a">CC Campaign</a></li>
+                    </ul>
+                    <div class="closeBtn">&times;</div>
+                  </div>
+            </nav>
+        </header>
+        <div class="homepage">
+            <h2 class="homepage-h2">"Hello! Sharing World"</h2>
+            <h1 class="homepage-h1">BEAUTY TECH AND INNOVATION SUMMIT <br> 2023</h1>
+            <p class="homepage-p">International event bringing together retail chains,distributors and beauty specialists.Discover the latest science,innovations and technologies that are revolutionizing the beauty and personal care industrywhile still retaining personalization.</p>
+            <h3 class="homepage-h3">2023.08.10(FRI) ~ 11(SAT) </h3>
+            <p class="homepage-p2">@ Bomas of Kenya Auditorium, Nairobi,Kenya.</p>
+        </div>
+    </section>
+
+    <!--Main Program-->
+    <section class="Program">
+        <h3 class="prog-h3">Main Program</h3>
+        <div class="underline"></div>
+        <ul class="prog-ul">
+            <li class="prog-li">
+                <img src="Logo/lectureicon.jpg" >
+                <h5 class="prog-h5"> Educate</h5>
+                <p class="prog-p">Learn how to identify tech solutions to your supply chain challenges and how to use tech in your formulations.</p>
+            </li>
+            <li class="prog-li">
+                <img src="Logo/program_icon_03 1.png">
+                <h5 class="prog-h5"> Innovate</h5>
+                <p class="prog-p">Utilize future technologies and innovations for both in-store and online retail.</p>
+            </li>
+            <li class="prog-li">
+                <img src="Logo/program_icon_04 1.png">
+                <h5 class="prog-h5"> Discover</h5>
+                <p class="prog-p">Discover the future of digital marketing capabilities and trends discovery using AI.</p>
+            </li>
+            <li class="prog-li">
+                <img src="Logo/connect.jpg">
+                <h5 class="prog-h5"> Connect</h5>
+                <p class="prog-p">Network with other professionals in the beauty industry and opportunities in the rise of holistic beauty.</p>
+            </li>
+            <li class="prog-li">
+                <img src="Logo/program_icon_02 1.png">
+                <h5 class="prog-h5"> Predict</h5>
+                <p class="prog-p">Effectively make predictions and driving consumer enagement using data.</p>
+            </li>
+         </ul>
+         <div class="button">
+            <button class="join-btn">Join Summit 2023</button>
+         </div>
+         <div class="text-btn">
+            <a href="#" class="join-text" title="See More button">SEE THE WHOLE PROGRAM</a>
+         </div>
+    </section>
+
+
+    <section class="featured-guests" id="featured-guests">
```

Comment:
- [ ] [REQUIRED ] Good job so far your contributor section is loading dynamically :clap:  But the design is a bit different compared to the original design. :call_me_hand:   Please make sure that this section has use the same font, `height`, `width `, and spacing `padding/margin `. If your `background color  is white, please change it  to the color defined on the original design  in order to make your  design similar to the original design :+1: 
 Check the screenshots below :point_down:  to see the differences.

#### Orignal design
![Capture](https://github.com/Cheptoo21/Capstone-Project-1/assets/6615394/342ad19d-887b-47ff-84dc-9443b7499271)


#### Your design
![Capture](https://github.com/Cheptoo21/Capstone-Project-1/assets/6615394/2be26bb7-3d8f-4f76-b408-ed38d126f8c0)